bin/mozetl-databricks.py

```diff
@@ -5,9 +5,48 @@
 import os
 import json
 import logging
+from base64 import b64encode
+from textwrap import dedent
 
 
-def run_submit(args, instance="dbc-caf9527b-e073.cloud.databricks.com"):
+def generate_runner(module_name, instance, token):
+    """Generate a runner for the current module to be run in Databricks."""
+
+    runner_data = """
+    # This runner has been auto-generated from mozilla/python_mozetl/bin/mozetl-databricks.py.
+    # Any changes made to the runner file will be over-written on subsequent runs.
+    from {module} import cli
+    
+    try:
+        cli.entry_point(auto_envvar_prefix="MOZETL")
+    except SystemExit:
+        # avoid calling sys.exit() in databricks
+        # http://click.palletsprojects.com/en/7.x/api/?highlight=auto_envvar_prefix#click.BaseCommand.main
+        pass
+    """.format(
+        module=module_name
+    )
+    logging.debug(dedent(runner_data))
+
+    request = {
+        "contents": b64encode(dedent(runner_data)),
+        "overwrite": True,
+        "path": "/FileStore/airflow/{module}_runner.py".format(module=module_name),
+    }
+    logging.debug(json.dumps(request, indent=2))
+    conn = httplib.HTTPSConnection(instance)
+    headers = {
+        "Authorization": "Bearer {token}".format(token=token),
+        "Content-Type": "application/json",
+    }
+    conn.request("POST", "/api/2.0/dbfs/put", json.dumps(request), headers)
+    resp = conn.getresponse()
+    logging.info("status: {} reason: {}".format(resp.status, resp.reason))
+    logging.info(resp.read())
+    resp.close()
+
+
+def run_submit(args):
```

Comment:
I think the `conn = httplib.HTTPSConnection(instance)` below now needs to be `conn = httplib.HTTPSConnection(args.instance)`?